pkg/kapis/tenant/v1alpha2/handler.go

```diff
@@ -15,15 +16,16 @@ type tenantHandler struct {
 	tenant tenant.Interface
 }
 
-func newTenantHandler(k8sClient k8s.Client, factory informers.InformerFactory) *tenantHandler {
+func newTenantHandler(_ k8s.Client, factory informers.InformerFactory) *tenantHandler {
```

Comment:
If parameter k8sClient is not used anymore, why not remove it from the function signature?

Reply:
done